app/components/avo/views/resource_show_component.html.erb

```diff
@@ -10,89 +10,116 @@
   <%= render Avo::PanelComponent.new(name: title, description: @resource.resource_description, display_breadcrumbs: @reflection.blank?, index: 0, data: { panel_id: "main" }) do |c| %>
```

Comment:
@adrianthedev this file was refactored in order to make first iteration of index on actions work. I found out we do not need it like this anymore, do you prefer to keep it like this or revert this changes?